framework/build.gradle

```diff
@@ -1,9 +1,11 @@
+ext {
+    annotatedJdkHome = '../../jdk'
+}
 sourceSets {
     main {
         resources {
             // Stub files, message.properties, etc.
-            srcDirs = ['src/main/java']
-            exclude '**/*.java'
+            srcDirs += ['src/main/java', "${buildDir}/generated/resources"]
```

Comment:
Will this now include both the `*.java` source files from `src/main/java` (which we don't want) and the annotated JDK source files (which we want)?
An update to the comment might make this clear.

Reply:
No, Gradle excludes *.java files from the source set automatically.